uefi/firmwarevolume.go

```diff
@@ -37,16 +37,16 @@ type Block struct {
 // FirmwareVolumeFixedHeader contains the fixed fields of a firmware volume
 // header
 type FirmwareVolumeFixedHeader struct {
-	Zeros          [16]uint8
+	Zeros          [16]uint8 `json:"-"`
```

Comment:
just fyi, the other way to do this I believe is just name it _. that's how I've done it in the past, then you don't the json gorp.
I prefer to just use _ in most cases because in one struct I run out of names like
unused1
unused2
etc. And the name _ makes it very clear you don't care.

Reply:
and it works.
https://play.golang.org/p/kttTW44tA3I

Reply:
done